mllib/src/main/scala/org/apache/spark/ml/stat/FValueTest.scala

```diff
@@ -63,11 +63,12 @@ object FValueTest {
    * @param dataset  DataFrame of continuous labels and continuous features.
    * @param featuresCol  Name of features column in dataset, of type `Vector` (`VectorUDT`)
    * @param labelCol  Name of label column in dataset, of any numerical type
-   * @return Array containing the FRegressionTestResult for every feature against the label.
+   * @return Array containing the FValueTestResult for every feature against the label.
```

Comment:
Likewise does the return value type actually change here or are you just fixing docs?

Reply:
I renamed ```FRegressionTestResult``` to ```FValueTestResult```, but forgot to change the docs here.